src/components/ContentSidebar/ActivityFeed/approval-comment-form/ApprovalCommentForm.scss

```diff
@@ -143,3 +148,14 @@
         }
     }
 }
+
+.pika-single {
```

Comment:
should this be scoped to .be?

Reply:
This part actually doesn't get scoped since it's dynamically added by the DatePicker BRUI component and is just a div attached to the body outside of sidebar itself

Reply:
Actually, this should not be done here. Add these styles to the pickaday component directly
https://git.dev.box.net/Box/box-react-ui/blob/master/src/components/date-picker/_pikaday.scss similar to how I did for others https://git.dev.box.net/Box/box-react-ui/pull/1186

Reply:
Made the change in BRUI `/box-react-ui/pull/1285`

Reply:
This PR will require a version upgrade of BRUI